core/frontend/src/IModelConnection.ts

```diff
@@ -69,10 +69,12 @@ export abstract class IModelConnection extends IModel {
   public readonly selectionSet: SelectionSet;
   /** The set of Tiles for this IModelConnection. */
   public readonly tiles: Tiles;
+  /** The set of [Category]($backend)'s in this IModelConnection. */
```

Comment:
@williamkbentley does the doc generator support some syntax for alternate display text in doc links? I seem to vaguely recall something like `[Display Name|Actual Name]($package)` but I found no evidence for its existence via grep.
Otherwise, pluralizing produces really awkward text as in this example ("Category's").